### DISCUSSION TITLE
Final Project Idea - Drone Safety and the Privacy/Transparency Tradeoff

### DISCUSSION THREAD

**Discussion by iamtrask** [OpenMined/courses#106] [General]
An applied version of the concepts in the course: https://www.theverge.com/2021/1/1/22209558/google-wing-faa-drone-remote-id-broadcast-rule-privacy-security

There's currently a big debate going on regarding whether the law should require all drones and drone operators to live-stream their location to the FAA whenever flying. 

This is EXACTLY the false choice between privacy and transparency we cover in the course - and a great example of how much of the world still has very little idea about better options than either "stream or don't stream". 

Detailing an example of how this could be done would be an *excellent* final project (although certainly very challenging).

Who knows - someone at the FAA might even come across it. :)

**Comment by physicsman** [comment]
[Flowstate : The FPV Drone Documentary (Full Trailer)](https://youtu.be/8TMUfvTD2qs)
[FAA Remote ID Final Ruling - Is it Truly Final?](https://www.youtube.com/watch?v=HoxD1eGeWF4)
